Feature/tracing with jaeger (#4)
distributed-technologies/services-jaeger: README.md

# services-jaeger
A umbrella helm chart for Jaeger.

## Jaeger components

> 

Reviewer: ```suggestion
```

Jaeger consist of 4 different components as shown in the picture below
![jaeger components](/images/Jaeger-components.PNG)
source: https://www.jaegertracing.io/docs/1.26/architecture/

### Query
This component contains a webserver that queries the database and show the result in a GUI 

### Storage

> the recommended storage option, and the one we are using is Elasticsearch as a document oriented database/engiene for us to store traces to make historic based analysis.

Reviewer: ```suggestion
the recommended storage option, and the one we are using is Elasticsearch as a document oriented database/engine for us to store traces to make historic based analysis.
```

### Collecter

> the collector is responsible for recieving the tracing data and store it to the database.

Reviewer: ```suggestion
The collector is responsible for receiving the tracing data and store it in database.
```

### Agent

> The agent is responsible for tracking the operation between services as described by the [OpenTracing specification](https://github.com/opentracing/specification/blob/master/specification.md)

Reviewer: ```suggestion
The agent is responsible for tracking the operations between services as described by the [OpenTracing specification](https://github.com/opentracing/specification/blob/master/specification.md)
```
